spec.md

```diff
@@ -190,6 +190,11 @@ If present, the value of this header MUST be a digest matching that of the respo
 
 If the blob is not found in the registry, the response code MUST be `404 Not Found`.
 
+A GET request may also include a `Range` request header to download part of a
+blob. The response code can either be `216 (Partial Content)` or `416 (Range
+Not Satisfiable)` in case of an invalid range. If the registry doesn't support
+range requests, it can respond with `Accept-Ranges: none`.
```

Comment:
```suggestion
Not Satisfiable)` in case of an invalid range. A registry MAY ignore the `Range` header.
```

Reply:
An older registry unaware of this change may ignore the header, but a newer registry may want to send a response back to discourage clients from send the Range request.